etc/windows/SyncThingSetup.nsi

```diff
@@ -71,7 +71,7 @@ Section "Command Line Interface" SecSyncThingCLI
 
   SetOutPath "$INSTDIR"
   
-  File /r "${SOURCEPATH}\syncthing-cli.exe"  
```

Comment:
syncthing-cli is not in fact the same thing as syncthing so I'm unsure what's the intention here. Your change may very well be valid - but are you sure?